### ISSUE TITLE
Position fixed for Tiered-menu not working

### ISSUE THREAD

**Issue by thornidlo** [primefaces/primeng#6700]
```
[x] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Plunkr Case (Bug Reports)**

https://stackblitz.com/edit/github-5mejcd-jehdsx?file=src/app/app.component.html

**Current behavior**
I have button with tieredmenu wrapped in fixed div (has position fixed). Need to open menu when click on button. Tieredmenu has default set position: absolute. If I open menu, it has right position under button. But if i scroll down/up, menu is still in that calculated position and moving when scrolling. This happens when appendTo='body' is set. This is logical, because menu is out of fixed div, where is button. if i remove appendTo='body' and scroll down/up, menu is show under button only when scroll is in top of page. If i scrolled down and click to open menu, it will show somewhere "in the air".

If i set class ui-tieredmenu position: fixed, menu will not showing, because on element is set display: none. This is not changed if i click on button.

**Expected behavior**
Correct working tiered-menu, when its position is set to fixed. 

* **Angular version:** 6.X

* **PrimeNG version:** 6.X

* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]
all

**Comment by JacobSiegle** [comment]
Is the demo working? Nothing is happening when I click the A-F buttons in Firefox or Chrome.

**Comment by thornidlo** [comment]
Yes, but that stackblitz with primeng menu component is wierd (maybe some setting missing). In console it created a html element, which has display: none, if you remove it, it will work.

**Comment by cagataycivici** [comment]
Please use forum for discussions and help requests.
